sudo-bmitch/version-bump
# dog-food version-bump on itself

Signed-off-by: Brandon Mitchell <git@bmitch.net>



### Fixes issue



### Describe the change

Use version-bump to update version-bump.



### How to verify it

```
$ make maint-version-check
CGO_ENABLED=0 go build -trimpath -ldflags "-s -w -extldflags -static -buildid=" -tags nolegacy -o bin/version-bump .
bin/version-bump check
processing file: .github/workflows/docker.yml for config .github/workflows/*.yml
processing file: .github/workflows/go.yml for config .github/workflows/*.yml
processing file: Dockerfile for config Dockerfile*
processing file: Dockerfile.buildkit for config Dockerfile*
```


### Changelog text



### Please verify and check that the pull request fulfills the following requirements
